lms/djangoapps/certificates/views/webview.py

```diff
@@ -254,7 +254,11 @@ def _update_course_context(request, context, course, platform_name):
     course_number = course.display_coursenumber if course.display_coursenumber else course.number
     context['course_number'] = course_number
     context['is_integrity_signature_enabled_for_course'] = settings.FEATURES.get('ENABLE_INTEGRITY_SIGNATURE')
-    if context['organization_long_name']:
+    context['idv_enabled_for_certificates'] = settings.FEATURES.get('ENABLE_CERTIFICATES_IDV_REQUIREMENT')
```

Comment:
Nit: I don't think we need this line.
From: e18e6af3961b7cacbc90c8a4160d3c47f99eca85